CODEOWNERS

```
@@ -0,0 +1,48 @@
+/ @cilium/committers
```

GitHub linter reports this needs to be fixed:

```suggestion
* @cilium/committers
```